Makefile

```diff
@@ -9,3 +14,19 @@ check:
 
 test: 
 	Rscript  -e "library(testthat); test()"
+
+docker_dev_image:
+	@docker run -it --name irkernel-build \
+	$(BASE_IMAGE) bash -c 'Rscript -e "install.packages(c('\''devtools'\''), repos = c('\''http://irkernel.github.io/'\'', '\''http://cran.rstudio.com'\''))" && \
+	                        Rscript -e "library(devtools)" \
+	                        -e "install_github('\''IRkernel/repr'\'')" \
+	                        -e "install_github('\''IRkernel/IRdisplay'\'')"'
+	@docker commit irkernel-build $(DEV_IMAGE)
+	@docker rm -f irkernel-build
```

Comment:
With modern docker (1.9+?), it's better to contain these steps in a Dockerfile.dev and simply `docker build -f Dockerfile.dev -t $(DEV_IMAGE) .`  to get the added value of caching.